src/zepben/evolve/streaming/grpc/connect.py

```diff
@@ -45,11 +45,35 @@ def _grpc_channel_builder_from_password(client_id: str, username: str, password:
 
 
 def connect_tls(host: str = "localhost", rpc_port: int = 50051) -> grpc.aio.Channel:
+    """
+    Connect to a Zepben gRPC service.
+
+    `host` The host to connect to.
+    `rpc_port` The gRPC port for host.
+
+    Returns a gRPC channel
+    """
     return _secure_grpc_channel_builder(host, rpc_port).build()
 
 
 def connect_with_password(client_id: str, username: str, password: str, host: str = "localhost", rpc_port: int = 50051,
                           conf_address: Optional[str] = None, **kwargs) -> grpc.aio.Channel:
+    """
+    Connect to a Zepben gRPC service.
+
+    `client_id` Your client id for your OAuth Auth provider.
+    `username` The username to use for an OAuth password grant.
+    `password` Corresponding password. If both `username` and `password` are provided, it takes precedence over the above client credentials.
+    `host` The host to connect to.
+    `rpc_port` The gRPC port for host.
+    `conf_address` The complete address for the auth configuration endpoint. This is used when an `token_fetcher` is not provided.
+        Defaults to http://<host>/auth
```

Comment:
so i just realised we don't actually default conf_address anymore - i think we need to change zepben-auth-python to actually try some default conf_addresses and then throw an exception if it doesn't work - at which point we should handle that exception here. 
This can be covered in EWB-1344 - I'll talk to you about it later.

Maybe for this PR you can change the documentation to say here: 
"Defaults to checking https://<host>/auth and https://<host>/ewb/auth"
and then we'll fix up the code after we make changes in zepben-auth-python